src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -36,6 +38,8 @@ public MaxwellSQLSyntaxError(String message) {
 
 	private final LinkedList<ColumnDef> columnDefs = new LinkedList<>();
 
+	private ArrayList<String> pkColumns;
```

Comment:
right, never mind this. I think null here is a valid value if there is no change in pks in the alter.